app/assets/stylesheets/forever_style_guide/modules/_nav.scss

```diff
@@ -1,96 +1,108 @@
+// Basic styles - A lot of overriding Bootstrap defaults
+.navbar-toggle {
+  @media (max-width: $grid-float-breakpoint) {
+    display: block;
+  }
+}
+.navbar-brand {
+  position: absolute;
+  width: 100%;
+  left: 0;
+  top: 0;
+  margin: auto;
+  padding: $padding-small-vertical 0 0;
+  text-align: center;
+}
+.navbar {
+  background-color: color('white');
+  border-radius: 0;
+  border: none;
+  @extend %card-shadow;
 
-      @media(max-width: $grid-float-breakpoint) {
-        @include border_highlight("left", transparent);
-      }
-    }
-    .nav-item-list {
-      @include border_highlight("left", transparent);
-    }
+  .nav.navbar-nav {
+    margin-top: 0;
+    margin-bottom: 0;
 
-    .nav-item-icon {
-      font-size: $font-size-400;
-      line-height: 20px;
-      vertical-align: bottom;
+    @media (max-width: $grid-float-breakpoint) {
+      margin-left: 0;
+      margin-right: 0;
     }
 
-    .nav-item-label {
-      line-height: 20px;
-    }
+    > li > a {
+      height: $header_height;
+      color: color('gray-400');
+      font-size: $font-size-500;
 
-    &:hover, &:focus {
-      .nav-item-tab,
-      .nav-item-list {
-        color: $color-nav-hover;
-        background-color: $color-nav-active;
+      @media (max-width: $grid-float-breakpoint) {
+        padding-top: $offcanvas-padding;
+        @include nav-border(top);
+        border-color: color('gray-600')
       }
-      .nav-item-tab {
-        @include border_highlight("bottom");
 
-        @media(max-width: $grid-float-breakpoint) {
-          @include border_highlight("left");
-        }
+      @media (min-width: $grid-float-breakpoint) {
+        color: color('gray-600');
+        font-size: $font-size-default;
+        @include transition(all 0.2s ease-in-out);
       }
-      .nav-item-list {
-        @include border_highlight("left");
+      &:hover {
+        @media (min-width: $grid-float-breakpoint) {
+          background-color: color('gray-300');
+        }
       }
     }
+    .dropdown.open > .dropdown-toggle {
+      color: color('white');
 
-    @at-root #{&}.active {
-      .nav-item-tab {
-        height: $header_height;
-        color: color('gray-600');
-        background-color: $color-nav-active;
-        @include border_highlight("bottom", color("primary"));
-        border-right: 1px solid color('gray-300');
-        border-left: 1px solid color('gray-300');
+      @media(min-width: $grid-float-breakpoint) {
+        color: color('gray-400');
+        background-color: $nav-bg-color;
+      }
+    }
+    > .active > a {
+      background-color: transparent;
 
+      &:hover,
+      &:focus {
         @media (max-width: $grid-float-breakpoint) {
-          @include border_highlight("left", color("primary"));
-          border-right: 1px solid color('gray-300');
-          border-bottom: 1px solid color('gray-300');
+          color: color('gray-400');
         }
       }
+    }
+    &.navbar-right {
+      margin-right: 0px;
```

Comment:
I'm not aware of cross-browser issues w/ untiless 0 values. The @mdo style guide states explicitly, 'Avoid specifying units for zero values, e.g., margin: 0; instead of margin: 0px;."  - http://codeguide.co/#css-syntax

The Mozilla Dev Network specifies that it's acceptable in certain circumstance, primarily when the 0 denotes length (https://developer.mozilla.org/en-US/Learn/CSS/Introduction_to_CSS/Values_and_units#Unitless_values)

I think the gist here is a 0 is a 0 is a 0, regardless of the units in play in your current styling scheme